slang-checks/src/main/java/org/sonarsource/slang/checks/CollapsibleIfStatementsCheck.java

```diff
@@ -31,7 +31,7 @@
 
 @Rule(key = "S1066")
 public class CollapsibleIfStatementsCheck implements SlangCheck {
-  private static final String MESSAGE_TEMPLATE = "Merge this %s statement with the nested one.";
+  private static final String MESSAGE_TEMPLATE = "Merge this \"%s\" statement with the nested one.";
```

Comment:
I did the same for some rule and used single quotes. We should have some style here:)

Reply:
Oh ok. I used double quote as the message for the other rule was also using double quotes.
I agree we should be consistent across messages... I created a [ticket](https://jira.sonarsource.com/browse/SONARSLANG-149) to harmonize this.